bin/linters.sh

```diff
@@ -70,16 +70,14 @@ go_metalinter() {
         --enable=goimports\
         --enable=golint --min-confidence=0 --exclude=.pb.go --exclude=pkg/config/proto/combined.go --exclude="should have a package comment"\
         --exclude=".*pkg/config/apiserver_test.go:.* method WriteHeaderAndJson should be WriteHeaderAndJSON"\
-        --enable=gosimple\
         --enable=ineffassign\
         --enable=interfacer\
         --enable=lll --line-length=160\
+        --enable=megacheck\
```

Comment:
found the answer while updating the linters in istio/istio (https://github.com/istio/istio/pull/443/files) it's a new warning:
```
WARNING: staticcheck, gosimple, and unused are all set, using megacheck instead
```